style.css

```diff
@@ -438,3 +439,188 @@ textarea {
   justify-content: left;
   cursor: pointer;
 }
+
+#navtag {
+  display: none;
+}
+
+/* Styling for desktop version of project */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    box-sizing: 0;
```

Comment:
- [OPTIONAL ] - Good job adding a media query to your project of this milestone +1 , but according to the project requirements, you should use a number between (768...1440)px instead of 768 px as the breakpoint between the desktop layout and another layout.